addon/components/denali-textarea.hbs

```diff
@@ -0,0 +1,10 @@
+{{! Copyright 2020, Verizon Media. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<div class="input
+  {{this.activeClass}}
+  {{this.warningClass}}"
+>
+  <textarea ...attributes>{{@value}}</textarea>
+  {{#if this.warningClass}}
+    <p class="message">{{this.warningMsg}}</p>
```

Comment:
Won't this render when there is no message given?

Reply:
<img width="275" alt="Screen Shot 2020-06-17 at 12 36 02 PM" src="https://user-images.githubusercontent.com/31903532/84930568-1e1d2c00-b097-11ea-8d62-f35465715eb9.png">
Looking at the dummy site this is what shows for my template. So it doesnt seem to be an issue. They have no <p> within the text area without the warning message.